prebid-server/developers/add-new-bidder-go.md

```diff
@@ -71,13 +71,14 @@ Please do not ignore errors from method calls made in your bid adapter code. Eve
 
 ### Bidder Info
 
-Let's begin with your adapter's bidder information YAML file. This file is required and contains your maintainer email address, specifies the ad formats your adapter will accept, and allows you to opt-in to video impression tracking.
+Let's begin with your adapter's bidder information YAML file. This file is required and contains your maintainer email address, your [GDPR Global Vendor List (GVL) id](https://iabeurope.eu/vendor-list-tcf-v2-0/), specifies the ad formats your adapter will accept, and allows you to opt-in to video impression tracking.
```

Comment:
I'd like to suggest we re-word this to let them opt-**out** of PBS video impression tracking.

Reply:
Sure.